base/_buttons.scss

```diff
@@ -0,0 +1,63 @@
+@charset "UTF-8";
+
+// buttons.scss
+// This file contains all styles related to the button element.
+// see http://bitters.bourbon.io
+// dependent on utilities/color.scss
```

Comment:
How about adding import-once as a dependency: https://github.com/Compass/compass/blob/master/import-once/README.md

That would save us from having to write conditional statement and would make the @import statement function more like I would expect it to. With import-once we should be able to declare the dependencies using import statement on each sass file without extra overhead or markup.